stdlog/file.c

```diff
@@ -80,7 +80,7 @@ static void
 file_open(stdlog_channel_t ch)
 {
 	if (ch->d.file.fd == -1) {
-		if((ch->d.file.fd = open(ch->d.file.name, O_WRONLY|O_CREAT|O_APPEND, 0660)) < 0)
+		if((ch->d.file.fd = open(ch->d.file.name, O_WRONLY|O_CREAT|O_APPEND|O_CLOEXEC, 0660)) < 0)
```

Comment:
CLOEXEC affects fd handling across execs, not across forks.  so if you have an app using liblogging and it forks and does its own processing, all of that continues to work.  what doesn't work is if you tried to exec a program and that program tried to write to the fd directly (because the handle would be closed automatically by the kernel).  although that design pattern is kind of broken in the first place as how would that program know what fd to even write to in the first place ?